src/platform-includes/sourcemaps/upload/cli/javascript.mdx

```diff
@@ -47,7 +39,108 @@ export SENTRY_ORG=___ORG_SLUG___
+
+```bash
+export SENTRY_FORCE_ARTIFACT_BUNDLES=1
+```
+
+### 2. Inject Debug IDs Using Sentry CLI
+
+Debug IDs require an additional but very important step, the injection. In this step, [`sentry-cli`](/product/cli/) will go over all the minified sources and their respective source maps by adding custom metadata that will be needed by Sentry while processing the event.
+
+<Note>
+
+Debug IDs injection will modify the content of both your minified sources and source maps which will create issues if you have integrity checks in place or you check for file changes before upload (e.g., on a CDN) based uniquely on their filename. _We are currently working on improving the injection system in order to circumvent these problems._
+
+</Note>
+
+In order to inject Debug IDs you will have to use the new `inject` command:
+
+```bash
+sentry-cli sourcemaps inject /path/to/file
+```
+
+<Note>
+
+You can verify your injected artifact files by looking at their contents and searching for `debugId` in the minified sources and `debug_id` in the source maps.
+
+</Note>
+
+### 3. Upload Artifacts Using Sentry CLI
+
+Once your artifact files are injected you can proceed to upload them to Sentry, with the `upload` command:
+
+```bash
+sentry-cli sourcemaps upload /path/to/file
+```
+
+<Note>
+
+You can verify your uploaded artifacts by navigating to **Project Settings -> Source Maps (left sidebar) -> Debug ID Bundles** on Sentry.
+
+</Note>
+
+<Note>
+
+Sentry CLI will notify you in case there are files that don't contain Debug IDs (e.g., you forgot to run the `inject` command). In that case, it will prompt you to weakly associate a release to the artifact. This weakly association is done in order to allow Sentry to properly symbolicate events without Debug IDs.
+
+</Note>
+
+#### Weak Release Association
+
+Since Debug IDs might cause issues for some users, we extended the new Debug IDs system with the concept of _Weak Release Association_. With such system you are able to tell Sentry to weakly associate the artifacts you upload with a `release`, and optionally `dist`.
+
+It is important that the name of the `release` you specified during upload will match exactly the name of the `release` in your SDK:
+```javascript
+Sentry.init({
+  // Make sure this value is identical to the name you give the release that specify during upload using Sentry CLI
+  release: "<release_name>",
+});
+```
+
+
+In order to associate an upload with a `release` and optionally `dist` you can add new parameters to the `upload` command:
+
+```bash
+sentry-cli sourcemaps upload --release=<release_name> --dist=<dist_name> /path/to/file
+```
+
+<Note>
+
+The difference between the old release-based system and the new system with weak release association is that in the new one we don't require you to create a release but you can just optimistically tell Sentry the name of your "future" release.
+
+</Note>
+
+## Upload with Release
```

Comment:
Do we want to call out in the title that this is the beginning of another (old) approach?
I just looked at how the document flows, and I can see people just copy-pasting commands and not realizing that they are done and they don't need to continue with this step - it's just another, older approach.

Reply:
You are indeed right, I will try to think of a way to make it apparent.